testsuite/cli/src/client_proxy.rs

```diff
@@ -673,43 +673,53 @@ impl ClientProxy {
         );
 
         // parse args
-        let (address, _) = self.get_account_address_from_parameter(space_delim_strings[1])?;
+        let (account, _) = self.get_account_address_from_parameter(space_delim_strings[1])?;
         let private_key = Ed25519PrivateKey::from_encoded_string(space_delim_strings[2])?;
         let consensus_public_key = Ed25519PublicKey::from_encoded_string(space_delim_strings[3])?;
-        let network_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[4])?;
-        let network_address = NetworkAddress::from_str(space_delim_strings[5])?;
-        let raw_network_address = RawNetworkAddress::try_from(&network_address)?;
-        let fullnode_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[6])?;
+        // let network_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[4])?;
+        let validator_network_address = NetworkAddress::from_str(space_delim_strings[5])?;
+        let validator_network_addresses = [validator_network_address];
+        // let fullnode_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[6])?;
         let fullnode_network_address = NetworkAddress::from_str(space_delim_strings[7])?;
-        let raw_fullnode_network_address = RawNetworkAddress::try_from(&fullnode_network_address)?;
+        let fullnode_network_addresses = [fullnode_network_address];
 
         let mut sender = Self::get_account_data_from_address(
             &mut self.client,
-            address,
+            account,
```

Comment:
It's naming confusion like this, that makes me wish that we just called it `Account` instead of `AccountAddress`